### ISSUE TITLE
What license to denote in package.json for proprietary packages?

### ISSUE THREAD

**Issue by leonadler** [npm/npm#17760]
*TL;DR: What value should be used in `package.json` to make npm not warn during `npm install`?*

#### I'm opening this issue because:

- [x] npm complains about the license in my package.json

Other closed issues do NOT resolve the issue: #8291, #8773, #8918, #10479, #15934

#### What's going wrong?

- I have a project which needs to be used on a propietary license for legal reasons
- I want to be able to install its dependencies and publish it via npm
- When running `npm install` is run, I get:
  ```
  npm WARN package-name@5.1.1 license should be a valid SPDX license expression
  ```
- The solutions provided in other issues do **NOT** work on npm 5.0.3
- Setting `"private": true` is **NOT** the solution.

#### How can the CLI team reproduce the problem?

1. Create a `package.json`:
  ```json
  {
    "name": "my-proprietary-package",
    "version": "0.0.1",
    "description": "npm issue example",
    "license": "UNLICENSED",
    "dependencies": {}
  }
  ```
2. Run `npm install`, get warning during install:
  ```
  npm WARN my-proprietary-package@0.0.1 license should be a valid SPDX license expression
  ```
3. Set `license` to `"SEE LICENSE IN LICENSE.md"`, add a `LICENSE.md` with any content.
4. Run `npm install` again, get warning during install:
  ```
  npm WARN my-proprietary-package@0.0.1 license should be a valid SPDX license expression
  ```
5. Set `license` to `"LicenseRef-LICENSE.md"`, run `npm install again`, get the same warning

**What value should be used in `package.json` to make npm not output the warning message?**

---

#### Supporting information:

npm 5.0.3, node 8.1.3, default registry, Linux
No proxy, Austria, Europe
